terra_layer/utils.py

```diff
@@ -21,8 +21,7 @@ def dict_merge(dct, merge_dct, add_keys=True):
 
 def get_layer_group_cache_key(pk, extras=[]):
     """
-    :param pk: The pk of the layer group to be cached
-    :type pk: int
+    :param scene: The scene to be cached
```

Comment:
the param doesn't match with function params

Reply:
I fix this